swagger.yaml

```diff
@@ -349,7 +349,7 @@ paths:
               type: string
               description: "Refresh token"
           schema:
-            $ref: '#/definitions/ExpirationTime'
+            $ref: '#/definitions/SignInExpirationTime'
```

Comment:
Ideally, you should create a new schema reference for the `password reset`, something like `PasswordResetExpirationTime` so that your endpoint swagger entry references that (can be identical to the SignInExpirationTime schema ref), which will give flexibilty later should the data being returned to the caller differ from both endpoints.